app/uk/gov/hmrc/pillar2/models/accountactivity/AccountActivitySuccess.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pillar2.models.accountactivity
+
+import play.api.libs.functional.syntax.given
+import play.api.libs.json.*
+
+import java.time.{LocalDate, LocalDateTime}
+
+case class AccountActivitySuccess(processingDate: LocalDateTime, transactions: Seq[AccountActivityTransaction])
+
+case class AccountActivityTransaction(
+  transactionType:   String,
+  transactionDesc:   String,
+  startDate:         Option[LocalDate],
+  endDate:           Option[LocalDate],
+  accruedInterest:   Option[BigDecimal],
+  chargeRefNo:       Option[String],
+  transactionDate:   LocalDate,
+  dueDate:           Option[LocalDate],
+  originalAmount:    BigDecimal,
+  outstandingAmount: Option[BigDecimal],
+  clearedAmount:     Option[BigDecimal],
+  standOverAmount:   Option[BigDecimal],
+  appealFlag:        Option[Boolean],
+  clearingDetails:   Option[Seq[AccountActivityClearance]]
+)
```

Comment:
I toyed around for a while with a more complicated model which split the charges, debits, and credits. But the schema doesn't give any guidance in this beyond the examples, and it would represent a bit of a deviation from the way the other endpoints in this codebase work. Plus, it's likely that model will eventually need to be extracted out if/when the backend is thinned out to better support audit event creation.

So I opted to keep this as anemic as possible for now.